src/sentry/incidents/subscription_processor.py

```diff
@@ -384,6 +386,17 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
             metrics.incr("incidents.alert_rules.skipping_already_processed_update")
             return
 
+        if features.has(
+            "organizations:workflow-engine-metric-alert-processing",
+            self.subscription.project.organization,
+        ):
+            # NOTE: feed the data through the new pipeline, but don't do anything with it yet.
+            # This will eventually change.
```

Comment:
🤔 what will change about this? seems 💯 to me. This will create the data packet from the subscription update and then send issue occurrences to the issue platform.